content/en/docs/security/contextual-authorization.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Security Contextual Authorization"
+linkTitle: "Contextual Authorization"
+description: "Contextual Authorization for securing Flux deployments."
+weight: 140
+---
+
+## Introduction
+
+Most cloud providers support context-based authorization, enabling applications
+to benefit from strong access controls applied to a given context (e.g. Virtual
+Machine), without the need of managing authentication tokens and credentials.
+
+For example, by granting a given Virtual Machine (or principal that such machine
+operates under) access to a AWS S3, applications running inside that machine can
+request a token on-demand, which would enable them access to the AWS S3 buckets
+without having to store long lived credentials anywhere.
+
+By leveraging such capability, Flux users can focus on the big picture, which is
+access controls enforcment with a least privileged approach, whilst not having to
+do deal with security hygine topics such as encrypting authentication secrets, and
+ensure they are being rotated regularly.
+All that is taken care automatically by the cloud providers, as the tokens provided
+are context and time bound.
+
+## Current Support
+
+Below is a list of Flux features that support this functionality and their documentation:
+
+| Status | Component | Feature | Provider | Ref |
+| ------ | ----------------- | ------- | -------- | ------ |
+| Supported | Source Controller | Bucket Repository Authentication | AWS | [Guide][AWS Buckets] |
+| Supported | Source Controller | Bucket Repository Authentication | Azure | [Guide][Azure Buckets] |
+| Supported | Source Controller | Bucket Repository Authentication | GCP | [Guide][GCP Buckets] |
+| Supported | Source Controller | OCI Repository Authentication | AWS | [Guide][AWS OCI Repository]  |
+| Supported | Source Controller | OCI Repository Authentication | Azure | [Guide][Azure OCI Repository]  |
+| Supported | Source Controller | OCI Repository Authentication | GCP | [Guide][GCP OCI Repository]  |
+| Supported | Image Reflector Controller | Container Registry Authentication | AWS | [Guide][AWS Container Registry] |
+| Supported | Image Reflector Controller | Container Registry Authentication | Azure | [Guide][Azure Container Registry] |
+| Supported | Image Reflector Controller | Container Registry Authentication | GCP | [Guide][GCP Container Registry] |
+| Supported | Kustomize Controller | SOPS Integration with Cloud KMS | AWS | [Guide][AWS KMS] |
+| Supported | Kustomize Controller | SOPS Integration with Cloud KMS | Azure | [Guide][Azure KMS] |
+| Supported | Kustomize Controller | SOPS Integration with Cloud KMS | GCP | [Guide][GCP KMS] |
+
+## Roadmap
+
+Support for context-based authorization should only increase over time.
```

Comment:
I wonder how we can expand this a little more and explicitly call out initiatives we're planning for the next couple of releases.

Reply:
That was my initial approach, but changed tact to just linking to the issue that tracks the progress instead.
Due to the GA priority, I am not sure this will be happening any time soon.